YourToDo/templates/registration/password_change_done.html

```diff
@@ -0,0 +1,11 @@
+{% extends "registration/registration_base.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Password changed" %}{% endblock %}
+
+{% block content %}
+<p>{% trans "Password successfully changed!" %}</p>
```

Comment:
Should we have a button to redirect users to the planner page or something else here?